website.py

```diff
@@ -40,6 +40,8 @@ def print_headers(content_type: str) -> None:
         cmd.append("--no-casts")
     if "allman" in form:
         cmd.append("--allman")
+    if "knr" in form:
```

Comment:
this either needs to be made an elif or the checkbox converted to a dropdown so you can't accidentally pass both options

Reply:
Sounds good, I don't really know web stuff so I'll just switch this to an elif.